src/pages/commits/CommitsWrapper.tsx

```diff
@@ -59,7 +59,10 @@ export const CommitsWrapper: React.FC<Props> = ({
             ) : (
               <ColumnContainer key={rolledUpVersions[0].id}>
                 <InactiveCommitLine />
-                <InactiveCommits rolledUpVersions={rolledUpVersions} />
+                <InactiveCommits
+                  hasFilters={hasTaskFilter}
```

Comment:
I'm a little confused as to why only the task filter triggers unmatching commits. When testing this I kinda expected that a build variant or status filter would also cause the unmatching label to appear.

Reply:
You are right. I made this change and added a check that includes all the filters. Nice Catch!